server/portal/templates/includes/nav_portal.raw.html

```diff
@@ -1,4 +1,6 @@
 {# @var user #}
+{# @var settings #}
```

Comment:
Optional anal suggestion:

```suggestion
{# @var user, settings #}
```

I am trying to keep the same number of lines in [templates cloned/tweak across multiple repos](https://confluence.tacc.utexas.edu/x/LoCnCQ) that I one day hope to consolidate.

Reply:
Makes them easier to compare.

Precedents:
- https://github.com/TACC/Core-CMS/blob/main/taccsite_cms/templates/header_branding.html
- https://github.com/TACC/Core-CMS/blob/main/taccsite_cms/templates/header_logo.html

Reply:
Ah, screw it.

[Markup linters are not gonna let me keep similar lines anyway.](https://github.com/TACC/tup-ui/actions/runs/3293274746/jobs/5429599502)

It was convenient while it lasted. But I've [documented the differences](https://confluence.tacc.utexas.edu/x/LoCnCQ) and they're close enough for me to resolve comfortably when the time comes.